CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# vNext
```

Comment:
[Nit] Better to tag it with next version 2.1.0, we can update it later on if there are some changes in between.